kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -24,6 +24,7 @@
     'apiserver_request_duration_seconds': 'request_duration_seconds',
     'apiserver_request_latencies': 'request_latencies',
     'apiserver_request_latency_seconds': 'request_latencies',
+    'process_cpu_seconds_total': 'process_cpu_seconds_total',
```

Comment:
I'm wondering if we should remove the `_seconds` part in the same since the info is already available in the metric metadata

```suggestion
    'process_cpu_seconds_total': 'process_cpu_total',
```